linera-service/src/node_service.rs

```diff
@@ -620,21 +620,20 @@ where
         &self,
         chain_id: ChainId,
         bytecode_id: BytecodeId,
+        block_effect_index: u32,
```

Comment:
This can't be something the user has to input (or think about) explicitly.

I understand that for now it's easier to have the client supply the application description as part of the operation. But the client still has to automatically compute it.